Dockerfile

```diff
@@ -2,20 +2,18 @@
 # supports Debian 11
 FROM python:3.8-slim-buster AS build-image
 
-RUN apt update
-RUN apt full-upgrade -y
-RUN apt install -y --no-install-recommends \
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
     build-essential \
     curl \
     default-libmysqlclient-dev \
     libpq-dev \
     unixodbc-dev \
     unzip
 
-ADD . /srcdir
+COPY . /srcdir
```

Comment:
why switch to `ADD`? `COPY` seems to be the best choice for copying files/directories locally, `ADD` is mostly meant to add support for remote copy, which is not needed here

Reply:
Exactly, switching to `COPY` is preferred. That's one of the things this PR is doing.

Reply:
Ah, I read the diff wrong, sorry